### DISCUSSION TITLE
better way to check error messages

### DISCUSSION THREAD

**Discussion by oeyoews** [williamboman/nvim-lsp-installer#811] [Q&A] [answered]
If lsp server installation fails with nvim-lsp-installer plugin, print error messages, check it with :messages instead of opening :lspinstallinfo to check error messages. Let me know if i'm wrong

**Comment by williamboman** [comment, marked as answer]
Hello! Yeah I think this would make sense. Due to how things are implemented now, installation output is however very tightly coupled with the floating window and would require quite significant changes. I just so happen to be rewriting things that would allow this to be a matter of a single configuration to control behavior, but this rewrite is so fundamental that it actually warrants a new plugin altogether. We'll see how things play out but for the time being this is not possible
